src/models/encrypted-model.test.ts

```diff
@@ -80,8 +82,8 @@ describe("Encrypted model", () => {
 		const id = newId();
 		await Dummy.create({ id, name: "test", a: "aaa1", b: "bbb1" });
 		const dummy = await Dummy.findOne({ where: { name: "test" } });
-		await dummy.decrypt("a");
-		await dummy.decrypt("b");
+		await dummy?.decrypt("a", getLogger("test"));
```

Comment:
This second arg is required. We missed this in a few places and it just wasn't being detected before.